docs/options api.md

```diff
@@ -127,6 +134,13 @@ elif self.options.logic == Logic.option_extreme:
     do_extreme_things()
 elif self.options.logic == "crazy":
     do_insane_things()
+
+# check if the current option is in a collection of integers using the class attributes
+if self.options.weapon in {Weapon.option_bow, Weapon.option_sword}:
+    do_stuff()
+# if we want to use a collection of strings a set won't work so use a tuple here
+elif self.options.weapon in ("none", "hammer"):
+    do_something_else()
```

Comment:
Given this footgun exists at all, would it be more intuitive to document `options.weapon.value in intset` and `options.weapon.current_key in stringset` instead? I understand _supporting_ `options.weapon in ...`, but I'm not sure if it makes sense to continue _recommending_ it in the docs when it forces us to go on this whole tangent about string sets.

Then again, I think recommending that the code specify whether it wants the option's int values or string values is just better style anyway? Is that just me?

Reply:
I'm not a big fan of accessing `Option.value`. I get the idea of being explicit and it's slightly faster but it still feels like an unnecessary specification when checking the option. maybe using current_key for the stringset is better since it will actually work, but i'm still not quite sure if that should be recommended

Reply:
I think if you want to be explicit here, you should have a third example that uses .value and a set. Something like
`# to make set work, we need to access the value directly, however in most cases using a tuple will suffice`